test_classify failure with dask-2021.12.0

Tests pass with dask-2021.11.2 but with dask-2021.12.0
```python
========================================== FAILURES ===========================================
___________________________________ test_equal_interval_cpu ___________________________________

    def test_equal_interval_cpu():
        k = 5
        # numpy
        numpy_ei = equal_interval(numpy_agg, k=5)
    
        unique_elements = np.unique(numpy_ei.data[da.isfinite(numpy_ei.data)])
        assert isinstance(numpy_ei.data, np.ndarray)
        assert len(unique_elements) == k
    
        # dask + numpy
>       dask_ei = equal_interval(dask_numpy_agg, k=k, name='dask_reclassify')

xrspatial/tests/test_classify.py:204: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
xrspatial/classify.py:936: in equal_interval
    out = _run_dask_numpy_equal_interval(agg.data, k)
xrspatial/classify.py:812: in _run_dask_numpy_equal_interval
    max_data = da.nanmax(data[da.isfinite(data)])
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

a = dask.array<getitem, shape=(nan,), dtype=float64, chunksize=(nan,), chunktype=numpy.ndarray>
axis = None, keepdims = False, split_every = None, out = None

    @derived_from(np)
    def nanmax(a, axis=None, keepdims=False, split_every=None, out=None):
        if np.isnan(a.size):
>           raise ValueError(f"Arrays chunk sizes are unknown. {unknown_chunk_message}")
E           ValueError: Arrays chunk sizes are unknown. 
E           
E           A possible solution: https://docs.dask.org/en/latest/array-chunks.html#unknown-chunks
E           Summary: to compute chunks sizes, use
E           
E              x.compute_chunk_sizes()  # for Dask Array `x`
E              ddf.to_dask_array(lengths=True)  # for Dask DataFrame `ddf`

../../.venv/xrspatial/lib/python3.8/site-packages/dask/array/reductions.py:548: ValueError
====================================== warnings summary =======================================
```

--- a/xrspatial/classify.py
+++ b/xrspatial/classify.py
@@ -809,8 +809,9 @@ def _run_numpy_equal_interval(data, k):
 
 
 def _run_dask_numpy_equal_interval(data, k):
-    max_data = da.nanmax(data[da.isfinite(data)])
-    min_data = da.nanmin(data[da.isfinite(data)])
+    data = da.where(data == np.inf, np.nan, data)
+    max_data = da.nanmax(data)
+    min_data = da.nanmin(data)
     width = (max_data - min_data) / k
     cuts = da.arange(min_data + width, max_data + width, width)
     l_cuts = cuts.shape[0]

